src/System.Management.Automation/utils/ClrFacade.cs

```diff
@@ -530,17 +530,21 @@ internal static object[] GetCustomAttributes<T>(Assembly assembly)
         #region Encoding
 
         /// <summary>
-        /// Facade for Encoding.Default
+        /// Facade for getting Encoding.Default
         /// </summary>
         internal static Encoding GetDefaultEncoding()
         {
             if (s_defaultEncoding == null)
             {
-#if CORECLR     // Encoding.Default is not in CoreCLR
-                // As suggested by CoreCLR team (tarekms), use latin1 (ISO-8859-1, CodePage 28591) as the default encoding.
-                // We will revisit this if it causes any failures when running tests on Core PS.
-                s_defaultEncoding = Encoding.GetEncoding(28591);
-#else
+#if UNUX        // PowerShell Core on Unix
+                s_defaultEncoding = Encoding.UTF8;
```

Comment:
I think it's a mistake to default to a UTF-8 encoding _with BOM_.
While .NET Core, at least currently, also reports the same BOM-based encoding - unofficially in `Encoding.Default` (not part of the contract yet) and officially in `Encoding.GetDefaultEncoding(0)` (and I consider that misguided too) -
at least it _doesn't govern the actual default behavior_ in the sense of _what happens if no encoding is specified_. The methods of type `System.IO.File` default to _BOM-less_ UTF-8 - both in .NET Framework since its inception and in .NET Core.
It think it's imperative that we use a _BOM-less_ UTF-8 encoding in the Unix world (and, I would argue, even make that the default on Windows (Core) as well, with opt-in to legacy behavior).